app/src/ui/resizable/resizable.tsx

```diff
@@ -113,7 +113,7 @@ export class Resizable extends React.Component<IResizableProps, {}> {
   }
 }
 
-export interface IResizableProps extends React.Props<Resizable> {
+export interface IResizableProps extends React.ClassAttributes<Resizable> {
```

Comment:
I think we should just drop the extends part here. This was from a time when we needed to get the `children` prop but these days that's automatically included by a type union.